src/software/geom/algorithms/convex_angle.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "software/geom/point.h"
+
+/**
+ * Calculates the convex angle formed by the two given vectors
+ * A convex angle is larger than 0 degrees and smaller than 180 degrees-
+ * https://proofwiki.org/wiki/Definition:Convex_Angle
```

Comment:
nit: To cover all scenarios, if two vectors are pointing in the same direction we return 0, and if they're pointing in opposite direction we return 180. This can be seen in the implementation given that we use `minDiff` and the documentation for `minDiff` states:
https://github.com/UBC-Thunderbots/Software/blob/c882e13b10b74cb7431d2834f67acc6cf7a1090e/src/software/geom/angle.h#L177
I understand that Convex angle is technically not inclusive of those angles, but I think it's better for the documentation to be more accurate of what we return, so I think we should update the doc to be more similar to the doc of `minDiff`. What do you think?

Reply:
Agreed- also, would it be better to update the documentation for convex_angle to use radians instead of degrees for consistency? We are using a unit-independent angle class so it should still be correct, right?